### ISSUE TITLE
Why huber loss is not averaged along minibatch axis?

### ISSUE THREAD

**Issue by fukatani** [pfnet/chainer#2415]
As `MeanSquaredError` and `MeanAbsoluteError`, huberloss should be averaged minibatch axis. Isn't it?

I mean
```python
return y.sum() / y.dtype.type(y.size),
```
Sorry if it is misunderstanding.

**Comment by delta2323** [comment]
Hi, your understanding is correct. So far as I remember and I read the original PR (#940), there was no discussion on it. I think it is purely because finer output was considered as better and we did not care the consistency between loss functions at that time.

We are discussing sample-wise output of `SigmoidCrossEntropy` (#2325) and `SoftmaxCrossEntropy` (#2357). It is welcome to take part in the discussion in it.

**Comment by fukatani** [comment]
Hi, thank you for your reply. :smile: 
Since it is better to unify into one issue from the viewpoint of consistency of the loss API, I close this issue.
